client/components/line.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import styles from './styles.js';
+import './css/app.css';
 
 const Line = (props) => {
-  return <hr style={styles.line}></hr>;
```

Comment:
I think you should've left line 5 alone; I would recommend reverting it (along w/ a lot of the `className` changes you made)